docs/tutorials/generic-plugin.md

```diff
@@ -0,0 +1,149 @@
+# Registering a generic plugin
```

Comment:
Here we we'll soon have a new functionality of automatically fetching the ABI data from etherscan according the `contractToCall`, then the user can choose from the contract write methods and their parameters. After filling the data, the encoded data that will be submitted is displayed on the screen.

It's not finished yet but it looks like this:
![Screen Shot 2020-10-01 at 14 19 29](https://user-images.githubusercontent.com/34843014/94803534-1d17cc80-03f2-11eb-94fb-9e85c498d97c.png)

![Screen Shot 2020-10-01 at 14 20 59](https://user-images.githubusercontent.com/34843014/94803543-22751700-03f2-11eb-845e-7974e8af49f3.png)

And if there is no ABI data available in the contract:
![Screen Shot 2020-10-01 at 14 21 50](https://user-images.githubusercontent.com/34843014/94803605-34ef5080-03f2-11eb-8cc1-f1e72deeff1c.png)